src/plugins/columnSorting/columnSorting.js

```diff
@@ -649,7 +649,7 @@ class ColumnSorting extends BasePlugin {
    * @returns {Number} Physical row index.
    */
   onModifyRow(row, source) {
-    if (this.blockPluginTranslation === false && source !== this.pluginName) {
+    if (this.blockPluginTranslation === false && source !== this.pluginName && this.isSorted()) {
       const rowInMapper = this.rowsMapper.getValueByIndex(row);
       row = rowInMapper === null ? row : rowInMapper;
```

Comment:
I think after change this check `rowInMapper === null` is always false, so it's useless.

Reply:
Good clue, but bigger indexes still enter into this `if` i.e. when `TrimRows` plugin is enabled.

Reply:
Ok. Thanks for checks.